mender/templates/gui-deploy.yaml

```diff
@@ -79,7 +79,7 @@ spec:
         - name: HAVE_MULTITENANT
           value: {{ .Values.global.enterprise | quote }}
         {{- end }}
-        {{- if .Values.auditlogs.enabled }}
+        {{- if and .Values.auditlogs.enabled .Values.global.enterprise }}
         - name: HAVE_AUDITLOGS
           value: {{ .Values.auditlogs.enabled | quote }}
```

Comment:
This variable doesn't make sense. If it is false then the whole block doesn't exist. so why not do `value: "true"` as in all the other cases?

Reply:
@bitcloud you are right, I force-pushed setting the value to `"true"`